package me.p0x38.fabric.client.mixins;

import net.minecraft.client.Camera;
import net.minecraft.client.renderer.GameRenderer;
import org.spongepowered.asm.mixin.Mixin;
import org.spongepowered.asm.mixin.gen.Invoker;

@Mixin(GameRenderer.class)
public interface GameRendererMixin {
    @Invoker("getFov")
    float fuckingUselessMod$getFov(
            Camera camera,
            float partialTicks,
            boolean applyEffects
    );
}